components/Grid/Item.tsx

```diff
@@ -26,32 +25,29 @@ const GridItem: React.FC<GridItemProps> = ({ item }) => {
     switch (item.api_model) {
       case "Work":
         setUrlPath("/items");
-        setEndpointPath("/works");
         setSupplementalInfo(item.work_type);
         return;
       case "Collection":
         setUrlPath("/collections");
-        setEndpointPath("/collections");
         return;
       default:
         setUrlPath("/items");
-        setEndpointPath("/works");
         setSupplementalInfo(item.work_type);
         return;
     }
   }, [item]);
 
-  if (!endpointPath) return <></>;
-
   return (
-    <ItemStyled key={item.id} data-item-id={item.id}>
+    <ItemStyled key={item.id} data-item-id={item.id} data-testid="grid-item">
       <Link href={`${urlPath}/${item.id}`}>
-        <a>
+        <a data-testid="grid-item-link">
           <Figure
             data={{
               aspectRatio: item.representative_file_set.aspect_ratio,
               isRestricted: isRestricted(item),
-              src: `${DCAPI_ENDPOINT}${endpointPath}/${item.id}/thumbnail`,
+              src: isFeatured
+                ? `${item.representative_file_set.url}/square/512,/0/default.jpg`
```

Comment:
Would this work? It'd route every Item/Figure src URI through our API endpoint, maybe clear things up for future usage with `next/image`? Not critical, just a thought.

```suggestion
                ? `${item.thumbnail}?aspect=square`
```